src/components/ToDoList/Todo.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { TodoPropsType } from './types';
+import EditTodoForm from './EditTodoForm';
+
+function Todo(props: TodoPropsType): JSX.Element {
+    const { task, deleteTodo, editTodo, editTask, toggleComplete } = props;
+
+    return (
+        <div className="flex relative justify-between">
+            <span className="flex items-center py-3 px-1 mb-4 cursor-grab text-purple-400">
+                <svg
```

Comment:
create src/assets and save svg in new folder

Reply:
done, but created coponents, because I need to pass props for some of them